Lib/test/test_descrtut.py

```diff
@@ -259,21 +259,21 @@ def merge(self, other):
     ...         print("classmethod", cls, y)
 
     >>> C.foo(1)
-    classmethod <class 'test.test_descrtut.C'> 1
+    classmethod C 1
```

Comment:
We want `classmethod test.test_descrtut.C 1`

Reply:
Turns out we don't: qualname is defined to not include the module name, so C is good here.

Alternative: change the code of the foo method to print `repr(cls)` instead of `cls`.